py_termianl_graph/py_terminal_graph.py

```diff
@@ -0,0 +1,131 @@
+
+
+
+class GraphStatic:
+	class Colors:
+		WHITE = "\033[0m"
+		BLACK = "\033[30m"  # "\033[90m"
+		RED = "\033[31m"
+		GREEN = "\033[32m"
+		ORANGE = "\033[33m"
+		BLUE = "\033[34m"
+		PURPLE = "\033[35m"
+		CYAN = "\033[36m"
+		LIGHT_GREY = "\033[37m"
+		DARK_GREY = "\033[90m"
+		LIGHT_RED = "\033[91m"
+		LIGHT_GREEN = "\033[92m"
+		YELLOW = "\033[93m"
+		LIGHT_BLUE = "\033[94m"
+		PINK = "\033[95m"
+		LIGHT_CYAN = "\033[96m"
+
+	def line(self, start=(0, 0), end=(10, 10), color=""):
+		try:
+			m = (end[1] - start[1]) / (end[0] - start[0])
+		except ZeroDivisionError as e:
+			return
+		b = start[1] - m * start[0]
+
+		for x in range(min(start[0], end[0]), max(start[0], end[0])):
+			if self.screen[int(m * x + b)][x] == " ":
+				self.screen[int(m * x + b)][x] = color + "█"
+
+
+	def __init__(self, data: list, color: str = "") -> None:
+		"""
+			Class that outputs simple graph by given data.
+			To draw a graph use GraphStatic.draw() method
+		:param data: list with tuple (float x, float y)
+		"""
+		from os import get_terminal_size
+
+		self.data = data
+		self.color = color
+
+		# Getting screen size
+		self.width, self.height = get_terminal_size()
+		self.width -= 1
+
+		# Creating matrix for screen
+		self.screen = [list(" " * self.width) for _ in range(self.height)]
+
+
+	def draw(self, color : str = ""):
+		"""
+			Process and draw an image
+		"""
+		self.get_image()
+		self.render()
+
+	def get_image(self):
+		"""
+			Draws all art to screen matrix
+		"""
+
+		from textwrap import TextWrapper  # Necessary for wrapping lines
+
+		left_margin = len(str(max([round(a, 5) for a, _ in self.data]))) + 1 # Distance between left edge and vertical line
+		bottom_margin = 5 # Distance between bottom corner and horizontal line (from bottom to top)
+
+		max_x = max([x for x, y in self.data])
+		max_y = max([y for x, y in self.data])
+
+		# ==== Operating Data =====
+		prev_pos = (left_margin, self.height - bottom_margin)
+		for num, (x, y) in enumerate(self.data): # Loop through all items in data
```

Comment:
I'm not sure, but I don't think this comment is helpful.
Even without the comment, the code is self-explanatory.

```suggestion
		for num, (x, y) in enumerate(self.data):
```